### PR TITLE
Put slow and disabled tests JSON in slow-disabled-tests branch and main.

### PR DESCRIPTION
**Pull request by kit1980**
First step for https://github.com/pytorch/test-infra/issues/175

### CODE REVIEW

File: .github/workflows/update_disabled_tests.yml [hunk 1]
```diff
@@ -20,6 +20,19 @@ jobs:
...
           cat disabled-tests.json
       - name: Push file to test-infra repository
+        uses: dmnemec/copy_file_to_another_repo_action@eebb594efdf52bc12e1b461988d7254322dac131
+        env:
+           API_TOKEN_GITHUB: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          source_file: 'disabled-tests.json'
+          destination_repo: 'pytorch/test-infra'
+          destination_folder: 'stats'
+          destination_branch: slow-disabled-tests
```
Comment (janeyx99) [inline .github/workflows/update_disabled_tests.yml:30]: Could we have a more general branch name for prosperity?

Comment (kit1980) [reply .github/workflows/update_disabled_tests.yml:30]: From this list which sounds better?
1. stats-update
2. generated-stats 
3. generated-files

Comment (janeyx99) [reply .github/workflows/update_disabled_tests.yml:30]: I like the second or first

**Review by janeyx99 — APPROVED** [review]
Approving conditional on the branch name change

**Pull request merged**